gpu4pyscf/pbc/dft/multigrid_v2.py

```diff
@@ -612,7 +637,13 @@ def evaluate_density_on_g_mesh(mydf, dm_kpts, kpts=None, xc_type='LDA'):
 
 
 def evaluate_xc_wrapper(pairs_info, xc_weights, img_phase):
```

Comment:
pass flag like `with_tau` or `xctype` to this function. and check the dimension of xc_weights accordingly.